galleries/users_explain/figure/figure_intro.rst

```diff
@@ -73,6 +73,15 @@ other than the default "inline" backend, you will likely need to use an ipython
 .. seealso::
     :ref:`interactive_figures`.
 
+.. note::
+
+   If you only need to use the classic notebook (i.e. `notebook<7`),
```

Comment:
```suggestion
   If you only need to use the classic notebook (i.e. ``notebook<7``),
```

If you want this to show up in code style, it needs to be double backtick, otherwise it treats it as a reference, which is failing the doc build.